packages/api/package.json

```
@@ -32,24 +32,30 @@
     "hashId:staging": "DOTENV_CONFIG_PATH=.env.staging ts-node -r dotenv/config scripts/url-hash.ts toHash",
     "unhashId": "ts-node -r dotenv/config scripts/url-hash.ts toId",
     "unhashId:prod": "DOTENV_CONFIG_PATH=.env.prod ts-node -r dotenv/config scripts/url-hash.ts toId",
-    "unhashId:staging": "DOTENV_CONFIG_PATH=.env.staging ts-node -r dotenv/config scripts/url-hash.ts toId"
+    "unhashId:staging": "DOTENV_CONFIG_PATH=.env.staging ts-node -r dotenv/config scripts/url-hash.ts toId",
+    "seed:config": "ts-node ../../node_modules/typeorm-seeding/dist/cli.js config",
+    "seed:run": "ts-node ../../node_modules/typeorm-seeding/dist/cli.js seed"
```

I think we could avoid defining specific paths by doing this.
```suggestion
    "typeorm-seeding": "ts-node $(yarn bin typeorm-seeding)",
    "seed:config": "yarn typeorm-seeding config",
    "seed:run": "yarn typeorm-seeding seed",
```